core/src/test/java/org/apache/calcite/test/JdbcTest.java

```diff
@@ -408,6 +409,18 @@ private static String q(String s) {
     }
   }
 
+  /**
+   * Adds table macro for connection.
+   */
```

Comment:
Illustrates the schema is with catalog "s" and the method reflection name is the name of the macro.